ci/before_deploy.bash

```diff
@@ -91,6 +96,15 @@ make_deb() {
     install -Dm644 README.md "$tempdir/usr/share/doc/$PROJECT_NAME/README.md"
     install -Dm644 LICENSE-MIT "$tempdir/usr/share/doc/$PROJECT_NAME/LICENSE-MIT"
     install -Dm644 LICENSE-APACHE "$tempdir/usr/share/doc/$PROJECT_NAME/LICENSE-APACHE"
+    cat > "$tempdir/usr/share/doc/$PROJECT_NAME/copyright" <<EOF
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: $PROJECT_NAME
+Source: $homepage
+
+Files: *
+Copyright: $maintainer
+License: Apache-2.0
```

Comment:
Is it possible to add the dual license here? `fd` is licensed under either of Apache or MIT.

Reply:
Ah ok, yes it is found how in https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/ applied in d5daae3948ada562e1ee8f7127b53a5de04f3715